src/app/home/upcoming-events/upcoming-events.component.html

```diff
@@ -0,0 +1,13 @@
+<div *ngIf="slides.length > 0" class="a">
+  <swiper-container
+    class="b"
+    [slidesPerView]="1.2"
+    [spaceBetween]="20"
+    [autoplay]="true"
+    [pagination]="{ clickable: true, dynamicBullets: true }"
+  >
+    <swiper-slide *ngFor="let slide of slides" class="c">
```

Comment:
Same here for the classes

Reply:
This is some serious season why i will never use "temporary" names for classes.